src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -351,16 +351,32 @@
    (comp first sql/result-set-seq)))
 
 (def store-catalogs-historically? (atom false))
+(def store-catalogs-jsonb-columns? (atom false))
```

Comment:
Can you put a comment here saying the same thing you did in your commit message about how these two are different?